Sources/Sentry/include/SentryLevelHelper.h

```diff
@@ -9,7 +9,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface SentryLevelHelper : NSObject
 
-+ (NSUInteger)breadcrumbLevel:(SentryBreadcrumb *)breadcrumb;
++ (NSString *_Nonnull)breadcrumbLevel:(SentryBreadcrumb *)breadcrumb;
```

Comment:
This is a breaking change as it's a public API.

I think it's best we add a new method. And use that internally instead. And we can annotate the old one as obsolete

Reply:
Make sense, I'll update it.

Reply:
I believe you're correct, it's not public. Either way, I would prefer if instead of modifying the signature, nameForSentryLevel were used inside SentryRRWebBreadcrumbEvent.init.

Reply:
Thanks for the suggestion, I've reverted the `breadcrumbLevel` changes and added the conversion to `SentryRRWebBreadcrumbEvent`.

[`c7e4835` (#4141)](https://github.com/getsentry/sentry-cocoa/pull/4141/commits/c7e483514e1d0e467e6c810de39f4a6bf75efe1d)